### PR TITLE
Add read write sheet

### PR DESCRIPTION
**Pull request by cameronbowie**

### CODE REVIEW

File: lib/smartsheet/api/body_builder.rb [hunk 1]
```diff
@@ -8,15 +10,21 @@ def initialize(endpoint_spec, request_spec)
       end
 
       def build
-            simple_body
+        json_body if endpoint_spec.sending_json?
```
Comment (armstnp) [inline lib/smartsheet/api/body_builder.rb]: If we're not sending JSON, should we just return the untouched body instead?

File: read-write-sheet/read_write_sheet.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+def row_update(row_id, column_id, value)
+  {
+      id: row_id,
+      cells: [{
+                  columnId: column_id,
+                  value: value
+              }]
+  }
```
Comment (armstnp) [inline read-write-sheet/read_write_sheet.rb]: The indentation here is kinda weird

Comment (cameronbowie) [reply read-write-sheet/read_write_sheet.rb]: Yep

File: test/smartsheet/api/body_builder_test.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    body = Smartsheet::API::BodyBuilder.new(@endpoint_spec, @request_spec)
+               .build
+
+    body.wont_be_nil
+    body.must_equal @some_body
+  end
+
+  it 'does not format nil JSON bodies' do
+    given_nil_json_body
+    body = Smartsheet::API::BodyBuilder.new(@endpoint_spec, @request_spec)
```
Comment (armstnp) [inline test/smartsheet/api/body_builder_test.rb]: If the same body is built in all tests, can its creation be extracted?

Comment (cameronbowie) [reply test/smartsheet/api/body_builder_test.rb]: Yes and done

**Pull request merged**